src/modules/auth/registerDevice.ts

```diff
@@ -6,11 +6,11 @@ import {
     performDuoPushVerification,
     performEmailTokenVerification,
     performTotpVerification,
-} from '../endpoints';
-import { askOtp, askToken, askVerificationMethod } from '../utils';
-import { getAuthenticationMethodsForDevice } from '../endpoints/getAuthenticationMethodsForDevice';
-import { requestEmailTokenVerification } from '../endpoints/requestEmailTokenVerification';
-import { SupportedAuthenticationMethod } from '../types';
+} from '../../endpoints';
+import { askOtp, askToken, askVerificationMethod } from '../../utils';
+import { getAuthenticationMethodsForDevice } from '../../endpoints/getAuthenticationMethodsForDevice';
+import { requestEmailTokenVerification } from '../../endpoints/requestEmailTokenVerification';
+import { SupportedAuthenticationMethod } from '../../types';
```

Comment:
you can add `import type` think about it because you have done it in other files.